babel.config.js

```diff
@@ -0,0 +1,36 @@
+module.exports = function(api) {
+  api.cache(true);
+
+  return {
+    presets: [
+      [
+        '@babel/preset-env',
+        {
+          modules: false,
+          targets: {
+            browsers: '> 1%',
```

Comment:
should we explicitly eclude IE 11 here, since we don't support it...? It's still included in the > 1% according to browserlist: https://browserl.ist/?q=%3E+1%25

Reply:
Good catch.